modules/ding_gatewayf/modules/ding_gatewayf_registration/templates/ding-gatewayf-registration-information.tpl.php

```diff
@@ -18,13 +18,11 @@
     <div class="pane-content">
       <div class="primary-content ding-gatewayf-registration-information">
         <h2 class="pane-title"><?php print render($title); ?></h2>
-        <div class="ding-gatewayf-registration-information--left">
+        <div class="long-text-wrapper">
```

Comment:
In the spirit of better class names should this be `ding-gatewayf--registration-information` and the parent class changed to `ding-gatewayf` (instead of `ding-gatewayf-registration-information`)  just as the class for the link is `ding-gatewayf--link-wrapper`?

Reply:
Class names will be changed.